File: eFrizer/flutter_efrizer/lib/models/credit_card/credit_card.dart
class CreditCard{
  final int creditCardId;
  final String cardName;
  final String expiryDate;
  final String cardHolderName;
  final String cvvCode;

  CreditCard({
    required this.creditCardId,
    required this.cardName,
    required this.expiryDate,
    required this.cardHolderName,
    required this.cvvCode
  });

  factory CreditCard.fromJson(Map<String, dynamic> json) {
    return CreditCard(
      creditCardId: int.parse(json['creditCardId'].toString()),
      cardName: json['cardNumber'],
      expiryDate: json['expiryDate'],
      cardHolderName: json['cardHolderName'],
      cvvCode: json['cvvCode'],
    );
  }

}

File: eFrizer/eFrizer.Model/CreditCard/CreditCard.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eFrizer.Model
{
    public class CreditCard
    {
        public int CreditCardId { get; set; }

        public string cardNumber { get; set; }
        public string expiryDate { get; set; }
        public string cardHolderName { get; set; }
        public string cvvCode { get; set; }

        public int ClientId { get; set; }

        public virtual Client Client { get; set; }
    }
}
